src/main/resources/view/DarkTheme.css

```diff
@@ -121,7 +121,7 @@
 }
 
 .cell_big_label {
-    -fx-font-family: "Segoe UI Semibold";
+    -fx-font-family: "Segoe UI";
```

Comment:
Hi @weixin-koh, just wanted to check which of these fields you're having problems manipulating. Changing the font family for `.cell_big_label` appears to work on my Mac. You might want to zero in on the font issue by checking if changing the font-size works as well.

Here's an external reference that I found with regards to integrating custom fonts - http://www.guigarage.com/2014/10/integrate-custom-fonts-javafx-application-using-css/

Reply:
Hi @sharan8, the one that I was having problems manipulating is actually the CSS font-family field for the HTML files, more specifically for `.cell_placeholder_web` as the font would appear to be Times New Roman when I specified it as "Segoe UI Semibold". However, I followed the external reference you gave and it works now! Thank you :)

Just curious though, it seems that the CSS for HTML files require an import of the font, but how is JavaFX able to access all my local fonts without importing? (I changed the `-fx-font-family` in `.cell_big_label` to a custom font I downloaded and it worked without having to import the font in the CSS file)

Reply:
Great that it works! @weixin-koh 

I'm not entirely sure how it may be able to access or interpret local fonts, and it could be a font that's present in JavaFX itself as well (you can check this by running `javafx.scene.text.Font.getFamilies();`). Regardless, do confirm if the font works as expected on your team members' workstations too. I'm guessing embedding the font files directly under resources would be best.

Using web fonts is another alternative that introduces a dependency, but works - http://fxexperience.com/2012/12/use-webgoogle-fonts-in-javafx/

Reply:
Yup I added the font file into the resources folder! I ran the line of code and it showed all of the fonts in my computer, so I think JavaFX just has a way to read in all your local fonts? Thanks so much for your help @sharan8!